hostagent.proto

```diff
@@ -64,6 +64,17 @@ message Command {
     message Install {
         string id = 1;
         optional string cloudinit = 2;  // cloudinit is the yaml configuration to optionally pass to an instance.
+        /*
+          Optional url to a WSL rootfs image we will download and import from. It will be registered as "id".
+          If this entry is not set, Ubuntu Pro for WSL will download a package from the Microsoft store matching WSL
+          distribution name as available in wsl --install.
+        */
+        optional Rootfs rootfs = 3;
+
+        message Rootfs {
+            string url = 1;
+            optional string sha256sum = 2;
```

Comment:
Should we mention that it should be base64 encoded?

Reply:
should it? there is no binary data to pass no?

Reply:
The checksum is binary

Reply:
The checksum is an hexadecimal string.

Reply:
I am probably missing something, but the checksum is generally hexadecimal as in https://pkg.go.dev/crypto/sha256#Sum256 example.

Reply:
The slice of bytes confused me :) Printing with "%s" shows strange output, but "%x" clarifies that it's just a hex-string.

Thanks for the clarification.